.github/workflows/tests-jellyfish.yml

```diff
@@ -32,15 +62,15 @@ jobs:
         with:
           node-version: '18'
 
-      - name: Populate environment
-        run: cd defichain && GIT_VERSION=1 ./make.sh ci-export-vars
-
-      - name: Build and setup
-        run: "cd defichain && docker build -t test-build-container
-          -f ./contrib/dockerfiles/x86_64-pc-linux-gnu.dockerfile ."
+      - name: Docker pull
+        run: |
+          docker pull ${{ env.DOCKER_IMAGE }}
 
       - name: Run tests
         run: |
           npm ci
           npm run compile
-          DEFICHAIN_DOCKER_IMAGE=test-build-container npm run ci:test -- --testTimeout=600000
+          DEFICHAIN_DOCKER_IMAGE=${{ env.DOCKER_IMAGE }} npm run ci:test -- --testTimeout=600000
```

Comment:
Might have to change passing defid as a docker image to instead running defid locally - dependent on the jellyfishsdk side of things